src/ResourceManager/Sql/Commands.Sql/Commands.Sql.csproj

```diff
@@ -462,13 +464,25 @@
       <HintPath>..\..\..\packages\Microsoft.Data.Services.Client.5.8.2\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@jiawang003 the references you are adding to this file (`Microsoft.Rest.ClientRuntime`, `Microsoft.Rest.ClientRuntime.Azure`, and `Newtonsoft.Json`) can be removed since they are already referenced using the `Common.Dependencies.targets` file found in the `tools` folder.

Reply:
removed those 3 refs